src/main/java/org/opensrp/repository/postgres/OrganizationRepositoryImpl.java

```diff
@@ -271,5 +278,35 @@ private Organization convert(org.opensrp.domain.postgres.Organization pgEntity)
 		pgOrganization.setParentId(organization.getPartOf());
 		return pgOrganization;
 	}
-
+	
+	@Override
+	public List<Organization> findSearchOrganizations(OrganizationSearchBean organizationSearchBean) {
+		Map<String, Integer> pageSizeAndOffset = getPageSizeAndOffset(organizationSearchBean);
+		return organizationMapper.selectSearchOrganizations(organizationSearchBean, pageSizeAndOffset.get("offset"),
+		    pageSizeAndOffset.get("pageSize"));
+	}
+	
+	private Map<String, Integer> getPageSizeAndOffset(OrganizationSearchBean organizationSearchBean) {
```

Comment:
why don't you change this return a pair of the 2 values instead of map. Map seems to be an overkill